controllers/gitrepository_controller.go

```diff
@@ -40,7 +40,7 @@ import (
 	"github.com/fluxcd/pkg/recorder"
 
 	sourcev1 "github.com/fluxcd/source-controller/api/v1alpha1"
-	intgit "github.com/fluxcd/source-controller/internal/git"
+	intgit "github.com/fluxcd/source-controller/pkg/git"
```

Comment:
```suggestion
	 "github.com/fluxcd/source-controller/pkg/git"
```

(or another alias that does not reflect "internal")